src/displayapp/screens/ScoreApp.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "Screen.h"
+#include "systemtask/SystemTask.h"
+#include "../LittleVgl.h"
+#include <lvgl/src/lv_core/lv_obj.h>
+
+namespace Pinetime::Applications::Screens {
+
+  class ScoreApp : public Screen {
+  public:
+    // TODO: Modes (tennis, other sports, magic the gathering/generic card game...)
+    // enum class Modes { Normal, Done };
+
+    ScoreApp(DisplayApp* app);
+
+    ~ScoreApp() override;
+
+    void Refresh() override;
+
+    void OnButtonEvent(lv_obj_t* obj, lv_event_t event);
+
+  private:
+    static constexpr uint16_t displayWidth = 240;
+    static constexpr uint16_t displayHeight = 240;
```

Comment:
`LV_HOR_RES` and `LV_VER_RES` already have these values, so they can be used in place of these.